src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -54,6 +54,7 @@
     <Compile Include="System\Runtime\Versioning\FrameworkName.cs" />
     <Compile Include="System\IO\PathTestsBase.cs" />
     <Compile Include="System\Net\WebUtility.cs" />
+    <Compile Include="System\Numerics\BitOperationsTests.cs" />
```

Comment:
Could you please move this to `'$(TargetGroup)'!='netstandard'` block to fix the netfx build flavor

Reply:
Thanks - fixed